octopusCatsEffect/src/main/scala/octopus/async/cats/cats.scala

```diff
@@ -0,0 +1,25 @@
+package octopus.async
+
+import _root_.cats.effect.IO
+import _root_.cats.implicits._
+import _root_.cats.Applicative
+
+import octopus.AppError
+
+package object cats {
+
+  implicit val catsIOAppError: AppError[IO] = new AppError[IO] {
+    override def pure[A](a: A): IO[A] = IO(a)
```

Comment:
let's get rid of overrides here - they're redundant

Reply:
Done